en/mkdocs.yml

```diff
@@ -268,9 +268,20 @@ nav:
     - 'EMail OTP Configurations': guides/mfa/email-otp-config-advanced.md
     - 'SMS OTP Configurations': guides/mfa/sms-otp-config-advanced.md
     - 'TOTP Configurations': guides/mfa/totp-config-advanced.md
+  - 'Admin Portal':
```

Comment:
Didn't we change the name to 'Developer Portal'

Reply:
Yes need to change it

Reply:
This will be updated in the future as we haven't change to admin portals name to developer portal yet.